augly/tests/image_tests/base_unit_test.py

```diff
@@ -103,7 +103,7 @@ def evaluate_class(
         dst = transform_class(
             self.img, metadata=metadata, bboxes=bboxes, bbox_format=bbox_format
         )
-
+        self.assertTrue(are_equal_images(self.img, dst))
```

Comment:
Be careful here -- we actually want to check that the **mode** of `self.img` & `dst` are equal, not that the entire images are equal. For example for `test_Crop` this would fail, because `dst` is the cropped image, which is smaller than `self.img` (the input image).
```suggestion
        self.assertTrue(self.img.mode == dst.mode)
```

Reply:
@zpapakipos how would this work for `convert_color()` since we are excluding it?